test/smoke/appServers/JBossEAP.gradle

```diff
@@ -1,41 +0,0 @@
-if (!project.hasProperty('jbossEapVersion')) {
```

Comment:
Why are we removing JBossEAP?

Reply:
Switching from testing JBossEAP to Wildfly so we can download without username/password, and seems better to use Wildfly anyways since that is what App Services supports